reference/7.3/Microsoft.PowerShell.Core/About/about_Language_Modes.md

```diff
@@ -14,66 +14,168 @@ Explains language modes and their effect on PowerShell sessions.
+`$ExecutionContext` variable.
+
+```powershell
+$ExecutionContext.SessionState.LanguageMode = "ConstrainedLanguage"
+```
+
+However, doing this is only useful for experimenting with language modes.
+Language modes are intended to provide added security to PowerShell sessions
+for specific contexts.
+
+Languages modes are set when you use a system application control policy or
+create a session configuration.
+
+### Using a system application control policy
+
+PowerShell automatically runs in `ConstrainedLanguage` mode when it's running
+under a system application control policy. The application control
+policies detected are AppLocker and Windows Defender Application Control (WDAC)
+on Windows platforms.
+
+PowerShell applies other restrictions besides language modes when it detects an
+application control policy. For example, there are additional restrictions to
+dot-sourcing and module importing under a policy.
+
+When a PowerShell session is started under a policy, it runs in
+`ConstrainedLanguage` mode. Beginning in PowerShell 7.4, the start up banner
+display includes a message indicating it's running in that mode. This allows
+users to have a usable interactive shell experience, running cmdlets and native
+commands, as well as access to basic language elements. But the user can't
+access PowerShell, .NET, or COM APIs that could be abused by a malicious actor.
+
+Any script or script-based module executed in this session runs in
+`ConstrainedLanguage` mode. However, any script or script-based module allowed
+by the policy runs in `FullLanguage` mode without any constraints. This way, a
+system locked down by policy can have scripts that are trusted by the policy
+and run with few constraints.
+
+### Using a session configuration
+
+PowerShell remoting optionally supports creating custom session configurations.
+configuration. You can set the language mode you want for that custom
+configuration. PowerShell Just Enough Administration (JEA) configurations use
+`NoLanguage` mode to restrict sessions to command invocation only. With JEA,
+the remote session can be restricted to specific users. The JEA users are
+limited to running a defined set of commands and can't directly access APIs,
+the file system, or other system resources.
+
+For more information, see [JEA Session configurations][01] and
+[New-PSSessionConfigurationFile][04].
 
-### Language modes
+## Language mode features and limitations
 
 This section describes the language modes in PowerShell sessions.
 
-#### Full language (FullLanguage)
+### FullLanguage mode
 
-The **FullLanguage** mode permits all language elements in the session.
-FullLanguage is the default language mode for default sessions on all versions
-of Windows except for Windows RT.
+The `FullLanguage` mode permits all language elements in the session.
+`FullLanguage` is the default language mode for default sessions on all
+versions of Windows.
```

Comment:
Is there a different language mode default for Linux/macOS? Given the new **ConfigurationFile** parameter, that might be worth clarifying in 7.3+ at least.

Reply:
No. The default is always `FullLanguage`. Policies only apply to Windows. Configurations can be used on any platform.